<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-VJRR3SJ" name="RetryMulitpleFailedTestCases" tests="12" failures="3" timestamp="28 Jan 2018 11:05:13 GMT" time="0.206" errors="0">
  <testcase name="test1" time="0.031" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.0" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.016" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.0" classname="Listners.DemoListner">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Listners.DemoListner.test1(DemoListner.java:12)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.0" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.015" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.0" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.0" classname="Listners.DemoListner">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Listners.DemoListner.test2(DemoListner.java:17)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test3" time="0.0" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.018" classname="Listners.DemoListner">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.001" classname="Listners.DemoListner">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Listners.DemoListner.test3(DemoListner.java:22)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- RetryMulitpleFailedTestCases -->
